apps/src/netsim/NetSimRemoteNodeSelectionPanel.html.ejs

```diff
@@ -2,8 +2,14 @@
 var utils = require('../utils');
 var _ = utils.getLodash();
 var i18n = require('./locale');
+var netsimGlobals = require('./netsimGlobals');
 var NodeType = require('./netsimConstants').NodeType;
 
+/** @type {function} */
+var assetUrl = netsimGlobals.getAssetUrlFunction();
```

Comment:
assetUrl --> getAssetUrl or makeAssetUrl? (I know this is not new, but I am just noticing it now)